documentation/how-to/attributes-form.en.md

```diff
@@ -31,8 +31,8 @@ The *editable* flag of fields is respected.
 
 ### Remember last values
 
-QField offers a much more fine-grained control over the last used values
-and ignores the QGIS setting.
+QField offers more fine-grained control over the last used values than QGIS,
+and uses its own settings instead of the QGIS ones.
```

Comment:
That's not really true these days. We also respect QGIS' setting (which was added a few versions ago).